### DISCUSSION TITLE
Forward to `mkdocs-material`

### DISCUSSION THREAD

**Discussion by ChristianKniep** [squidfunk/mkdocs-material#3401] [General]
I really like the forward to a specific page to not have the menue visible. But I am not sure how that is done browsing the repository.
When opening the website locally (after `mkdocs serve`) I am forwarded.

How is this controlled?

```
WARNING  -  [19:30:39] "GET / HTTP/1.1" code 302
INFO     -  [19:30:39] Browser connected: http://localhost:8000/mkdocs-material/
```

**Comment by squidfunk** [comment]
The development server of MkDocs does the 302 redirect. You may ask the maintainers of MkDocs if you want to know more about this functionality, as we're only a downstream project which uses MkDocs.
